src/Styles/index.css

```diff
@@ -0,0 +1,92 @@
+@import "./star.css";
+@import "./button.css";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
That's better to spread these styles over several files according to their dedication